FileIO/Legacy/MeshIO.cpp

```diff
@@ -133,18 +148,22 @@ MeshLib::Mesh* MeshIO::loadMeshFromFile(const std::string& file_name)
 	}
 }
 
-MeshLib::Element* MeshIO::readElement(const std::string& line,
-                                      const std::vector<MeshLib::Node*> &nodes)
+std::size_t MeshIO::readMaterialID(std::istream & in) const
+{
+	unsigned index, patch_index;
```

Comment:
Afaik GMSH uses patch_index as material id. But you are right: for readability the variable should be renamed.